membrane_curvature/base.py

```diff
@@ -128,6 +142,16 @@ def __init__(self, universe, select='all',
                 warnings.warn(msg)
                 logger.warn(msg)
 
+        # Apply wrapping coordinates
+        if self.wrap:
+            self.ag.wrap()
+        else:
+            msg = (" `wrap == False` may result in inaccurate calculation "
+                   "of membrane curvature. Surfaces will be derived from "
+                   "a reduced number of atoms.")
+            warnings.warn(msg)
+            logger.warn(msg)
```

Comment:
Can you please add in something about ignoring this warning if the trajectory is processed to rotate -- otherwise, people who are following the directions and not using `wrap=True` will be unnecessarily alarmed.